test/lib/code_corps_web/controllers/conversation_controller_test.exs

```diff
@@ -68,13 +68,16 @@ defmodule CodeCorpsWeb.ConversationControllerTest do
 
   describe "update" do
     @tag authenticated: :admin
-    test "updates and renders chosen resource when data is valid", %{conn: conn, current_user: current_user} do
-      message = insert(:message)
-      conversation = insert(:conversation, user: current_user, message: message)
+    test "updates and renders chosen resource when data is valid", %{conn: conn, current_user: user} do
+      %{project: project} = insert(:project_user, role: "admin", user: user)
+      message_on_user_administered_project = insert(:message, project: project)
+      conversation_on_user_administered_project =
+        insert(:conversation, message: message_on_user_administered_project)
+      assert conn |> request_update(conversation_on_user_administered_project, %{status: "wat"}) |> json_response(422)
```

Comment:
It looks like you're doing a `422` assert in here. This line probably needs removed.